modules/serverless-common/pages/service-mesh/common-service-mesh-setup.adoc

```diff
@@ -1,28 +1,82 @@
-= Setup Eventing with {SMProductName}
+= Setup {serverlessproductname} with {SMProductName}
 :compat-mode!:
 // Metadata:
-:description: Setup Eventing with {SMProductName}
+:description: Setup {serverlessproductname} with {SMProductName}
 
 // TODO
 
+.Assumptions and limitations
+
+This page describes the integration of {serverlessproductname} with {smproductname}. In this setup, it is assumed that:
+
+* All Knative internal components, as well as Knative Services, are part of the {smproductshortname} and have istio-sidecars injected. This means, that strict mTLS is enforced within the whole mesh. All requests to Knative Services require an mTLS connection  (the client must send its certificate), except calls coming from OpenShift Routing.
+* {serverlessproductname} with Kourier disabled can only target *one* istio service mesh. Multiple meshes can be present in the cluster, but {serverlessproductname} will only be available on one of them.
+* The mesh that {serverlessproductname} is part of has to be distinct and ideally only for {serverlessproductname} workloads, as additional configuration like `Gateways` might interfere with the automated configuration by {serverlessproductname}. Other namespaces can be part of this Mesh, but must not configure the istio custom resource `Gateway` that could overlap with {serverlessproductname} `knative-local-gateway` and `knative-ingress-gateway`.
+* {smproductname} only allows one `Gateway` to claim a wildcard host binding (`hosts: *`) on the same port (`port: 443`). So the `knative-ingress-gateway` and `knative-local-gateway` have to be unique in the Mesh. If another Mesh is already binding this configuration, a separate Mesh has to be created for {serverlessproductname} workloads.
+* Cluster external Knative Services are expected to be called via OpenShift Ingress using OpenShift Routes. It is not supported to access istio directly, e.g. by exposing the `istio-ingressgateway` using a Service with type `NodePort` or `LoadBalancer`.
+* Changing the target `ServiceMeshMemberRole`, that {serverlessproductname} is part of (meaning moving {serverlessproductname} to another mesh) is not supported. The only way to change the targeted Service Mesh is to uninstall and reinstall {serverlessproductname}.
+
+
 .Prerequisites
 
 * You have access to an {product-title} account with cluster administrator access.
 
 * You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
```

Comment:
Same comment as above, we should define the notion of a tenant and then say that you need access to all those tenant projects. Give them names here, too (`tenant-1`, `tenant-2`, ...)

Reply:
Tenants are not part of that documentation here. This is just defining your mesh. I dropped that bullet entirely, as it is not really relevant.